### PR TITLE
Edit tree lemmatizer

### PR DESCRIPTION
**Pull request by qeterme**
Replaced Lemmy lemmatizer in the large model with the new Edit Tree Lemmatizer

### CODE REVIEW

**Comment by oroszgy** [comment]
Looks good, thanks! How does the lemmatization accuracy changes?

**Comment by qeterme** [comment]
On UD-Szeged it now performs 97.03%, which is up 1 percent from Lemmy.

File: hu_core_news_lg/configs/tagger.cfg [hunk 1]
```diff
@@ -103,7 +120,7 @@ gpu_allocator = ${system.gpu_allocator}
 dropout = 0.1
 accumulate_gradient = 1
 patience = 5000
-max_epochs = 15
```
[the remark this answers is not in the document]
Comment (oroszgy) [reply hu_core_news_lg/configs/tagger.cfg:106]: 7 should be fine

**Comment by oroszgy** [comment]
Nice job, thanks!

**Pull request merged**